docs/client/javascript-mono/_reactInstall.mdx

```diff
@@ -1,12 +1,13 @@
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+Note since the `@statsig/react-bindings` works in conjuction with `@statsig/js-client`, 
+the [@statsig/js-client documentation](/client/javascript-sdk) can also be relevant for React implementations.
 
```

Comment:
this isnt sufficient - we need inline docs on initialization behaviors for react, react native, expo, etc